### PR TITLE
Refresh layer widgets

### LINKED ISSUE #743: When a filter changes, only update the widgets associated to the same layer
Right now we're refreshing all widgets.

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Fixes #743.

@xavijam what you do you think?

### CODE REVIEW

File: src/windshaft/client.js [hunk 1]
```diff
@@ -22,19 +22,21 @@ cdb.windshaft.Client.MAX_GET_SIZE = 2033;
  * @param  {function} callback A callback that will get the public or private map
  * @return {cdb.windshaft.DashboardInstance} The instance of the dashboard
  */
-cdb.windshaft.Client.prototype.instantiateMap = function(mapDefinition, filters) {
+cdb.windshaft.Client.prototype.instantiateMap = function(options) {
+  var mapDefinition = options.mapDefinition;
```
Comment (xavijam) [inline src/windshaft/client.js]: Finally changed the way to report error or success :).

Comment (alonsogarciapablo) [inline src/windshaft/client.js]: Yes! I like it!

File: src/windshaft/dashboard.js [hunk 2]
```diff
@@ -12,66 +12,79 @@ cdb.windshaft.Dashboard = function(options) {
...
 
-    // TODO: Could be https
-    var url = widgetMetadata.url.http;
-    widgetModel.set({
+    widget.set({
       'url': url,
-      'boundingBox': self.boundingBoxFilter.toString()
+      'boundingBox': boundingBoxFilter.toString()
+    }, {
+      silent: silent
```
Comment (xavijam) [inline src/windshaft/dashboard.js]: It looks a good solution :)-

**Comment by xavijam** [comment]
Fine by me

**Pull request merged**